pkgs/applications/gis/gol/default.nix

```
@@ -0,0 +1,44 @@
+{ lib, maven, fetchFromGitHub, makeWrapper, jre }:
+
+maven.buildMavenPackage rec {
+  pname = "gol";
+  version = "0.1.9";
+
+  src = fetchFromGitHub {
+    owner = "clarisma";
+    repo = "gol-tool";
+    rev = "${version}";
```

Don't need to wrap it in a string
```suggestion
    rev = version;
```

---

Thanks! Fixed